_dev/src/ts/pages/RestorePageBackupSelection.ts

```diff
@@ -0,0 +1,58 @@
+import StepPage from './StepPage';
+import api from '../api/RequestHandler';
+
+export default class RestorePageBackupSelection extends StepPage {
+  protected stepCode = 'backup-selection';
+
+  constructor() {
+    super();
+  }
+
+  public mount = () => {
+    this.initStepper();
+    this.#form.addEventListener('change', this.#saveForm.bind(this));
+    this.#form.addEventListener('submit', this.#handleSubmit);
+  };
+
+  public beforeDestroy = () => {
+    this.#form.removeEventListener('change', this.#saveForm.bind(this));
+    this.#form.removeEventListener('submit', this.#handleSubmit);
+  };
+
+  get #form(): HTMLFormElement {
+    return document.forms.namedItem('backup_choice')!;
+  }
+
+  #saveForm = async () => {
+    const routeToSave = this.#form!.dataset.routeToSave;
```

Comment:
```suggestion
    const routeToSave = this.#form.dataset.routeToSave;
```

Based on the type of the getter, `#form` should be always defined.